EnglishFilter/sections/foreign.txt

```diff
@@ -8577,6 +8577,10 @@ league-funny.com##.widgetBox.w728Bottom
 league-funny.com##.right > .fansCard
 ! https://github.com/AdguardTeam/AdguardFilters/issues/35871
 ck101.com#?##threadlisttableid > tbody:not([class]):not([id]):has(> tr > td[id^="popin_"])
+! https://github.com/AdguardTeam/AdguardFilters/issues/75538
+xs26.com##body > div[class^="___"]
+xs26.com##.show-app2
+xs26.com##.nr_page > div[id] > ul
```

Comment:
Not sure about al these rules. Ads are blocked, and was injected by third-party scripts.
Also the site is Chinese. We have this filter.

Reply:
<details><summary>Screenshot:</summary>

![image](https://user-images.githubusercontent.com/68852737/109629119-90b23700-7b54-11eb-9ad1-939eef894ba0.png)
</details><br/>
with userconfig.

Reply:
better block ad domains